docs/develop/support-ie-11.md

```diff
@@ -1,27 +1,31 @@
 ---
-title: Support Internet Explorer 11
-description: Learn how to support Internet Explorer 11 and ES5 Javascript in your add-in.
-ms.date: 05/01/2022
+title: Support Internet Explorer 11 and Microsoft Edge Legacy
+description: Learn how to support Internet Explorer 11 and Microsoft Edge Legacy in your add-in.
+ms.date: 12/12/2022
 ms.localizationpriority: medium
 ---
 
-# Support Internet Explorer 11
+# Support Internet Explorer 11 and Microsoft Edge Legacy
 
 > [!IMPORTANT]
-> **Internet Explorer still used in Office Add-ins**
+> **Internet Explorer and Microsoft Edge Legacy still used in Office Add-ins**
 >
-> Some combinations of platforms and Office versions, including perpetual versions through Office 2019, still use the webview control that comes with Internet Explorer 11 to host add-ins, as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). We recommend (but don't require) that you continue to support these combinations, at least in a minimal way, by providing users of your add-in a graceful failure message when your add-in is launched in the Internet Explorer webview. Keep these additional points in mind:
+> Some combinations of platforms and Office versions, including perpetual versions through Office 2019, still use the webview controls that come with Internet Explorer 11 and Microsoft Edge Legacy (EdgeHTML-based) to host add-ins, as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). We recommend (but don't require) that you continue to support these combinations, at least in a minimal way, by providing users of your add-in a graceful failure message when your add-in is launched in these webviews. Keep these additional points in mind:
 >
-> - Office on the web no longer opens in Internet Explorer. Consequently, [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) no longer tests add-ins in Office on the web using Internet Explorer as the browser.
-> - AppSource still tests for combinations of platform and Office *desktop* versions that use Internet Explorer, however it only issues a warning when the add-in does not support Internet Explorer; the add-in is not rejected by AppSource.
+> - [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) no longer tests add-ins in Office on the web using Internet Explorer or Microsoft Edge Legacy. Office on the web no longer opens in Internet Explorer.
```

Comment:
Can someone have Edge Legacy as a browser anymore? These articles are written from a browser-centric perspective, even though it's really the Office version that's dictating the webview.

Reply:
Good point. Even if Edge Legacy were somehow still installed on older Windows OS versions, such as Windows 7 and 8/8.1, users would be prompted to update to the Chromium-based version to be able to use the browser. I'll update line 15 (and the other article where this is referenced) to include Edge Legacy.